models/nequip/test_nequip_kappa.py

```
@@ -0,0 +1,348 @@
+    compiled_model_file = next(iter(matching_files))
+elif os.path.exists(f"{compile_path}"):
+    compiled_model_file = f"{compile_path}"
+else:
+    raise FileNotFoundError(f"No compiled model file was not found at {compile_path}!")
+
+
+def calc_kappa_for_structure(
+    *,  # force keyword-only arguments
+    atoms: Atoms,
+    displacement_distance: float,
+    compile_path: str,
```

Ruff (0.12.2)

Unused function argument: `compile_path`

(ARG001)